src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+
+  --color-mockup-blue: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo__link {
+  cursor: pointer;
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  height: max-content;
+}
+
+.nav__link {
+  display: inline-block;
+  text-decoration: none;
+  position: relative;
+  font-family: inherit;
+  color: #000;
+  text-transform: uppercase;
+  padding: 23px 0;
+  margin-left: 20px;
+}
+
+.nav__link:hover {
+  color: var(--color-mockup-blue);
+}
+
+.is-active {
+  color: var(--color-mockup-blue);
+  margin-left: 0;
```

Comment:
move this to `:first-child` bc `.is-active` may be not first element

Reply:
everything is fine, thanks for the fix, but there is one error left
![image](https://user-images.githubusercontent.com/116076758/208385035-4d8e8031-b35c-499a-9d07-8d55e457de80.png)